### ISSUE TITLE
Explaining Deep Learning Models using Causal Inference

### ISSUE THREAD

**Issue by pkfancy** [dais-ita/interpretability-papers#64]
@misc{1811.03804,
Author = {Simon S. Du and Jason D. Lee and Haochuan Li and Liwei Wang and Xiyu Zhai},
Title = {Gradient Descent Finds Global Minima of Deep Neural Networks},
Year = {2018},
Eprint = {arXiv:1811.03804},
}
